src/annotator/test/guest-test.js

```diff
@@ -262,55 +279,82 @@ describe('Guest', () => {
       it('scrolls to the anchor with the matching tag', () => {
         const highlight = document.createElement('span');
         const guest = createGuest();
+        const fakeRange = sinon.stub();
         guest.anchors = [
-          { annotation: { $tag: 'tag1' }, highlights: [highlight] },
+          {
+            annotation: { $tag: 'tag1' },
+            highlights: [highlight],
+            range: new FakeTextRange(fakeRange),
+          },
         ];
+
         emitGuestEvent('scrollToAnnotation', 'tag1');
+
         assert.called(scrollIntoView);
         assert.calledWith(scrollIntoView, highlight);
       });
 
-      context('when dispatching the "scrolltorange" event', () => {
-        it('emits with the range', () => {
-          const highlight = document.createElement('span');
-          const guest = createGuest();
-          const fakeRange = sinon.stub();
-          guest.anchors = [
-            {
-              annotation: { $tag: 'tag1' },
-              highlights: [highlight],
-              range: fakeRange,
-            },
-          ];
-
-          return new Promise(resolve => {
-            guest.element.addEventListener('scrolltorange', event => {
-              assert.equal(event.detail, fakeRange);
-              resolve();
-            });
+      it('emits a "scrolltorange" DOM event', () => {
+        const highlight = document.createElement('span');
+        const guest = createGuest();
+        const fakeRange = sinon.stub();
+        guest.anchors = [
+          {
+            annotation: { $tag: 'tag1' },
+            highlights: [highlight],
+            range: new FakeTextRange(fakeRange),
+          },
+        ];
 
-            emitGuestEvent('scrollToAnnotation', 'tag1');
+        return new Promise(resolve => {
+          guest.element.addEventListener('scrolltorange', event => {
+            assert.equal(event.detail, fakeRange);
+            resolve();
           });
+
+          emitGuestEvent('scrollToAnnotation', 'tag1');
         });
+      });
 
-        it('allows the default scroll behaviour to be prevented', () => {
-          const highlight = document.createElement('span');
-          const guest = createGuest();
-          const fakeRange = sinon.stub();
-          guest.anchors = [
-            {
-              annotation: { $tag: 'tag1' },
-              highlights: [highlight],
-              range: fakeRange,
+      it('allows the default scroll behaviour to be prevented', () => {
```

Comment:
But why? What is the meaning of that? (I know you inherited this test description).

Reply:
I don't know - as you've noticed, this is just inherited code. Unfortunately it is an exposed API so we can't just remove it without doing the work to confirm nobody is using it.